### PR TITLE
Update Polish translation

### PR DESCRIPTION
**Pull request by alewicki95**
Added missing Polish translation

### CODE REVIEW

File: po/pl.po [hunk 1]
```diff
@@ -126,23 +126,23 @@ msgstr "Położenie panelu na ekranie"
 
 #: prefs.js:443
 msgid "Stacked to top"
-msgstr ""
+msgstr "Przypomnij do góry"
```
Comment (rathann) [inline po/pl.po:129]: Typo: "Przypnij". Perhaps "Przypięte" is also an option.

File: po/pl.po [hunk 2]
```diff
@@ -126,23 +126,23 @@ msgstr "Położenie panelu na ekranie"
...
 
 #: prefs.js:443
 msgid "Stacked to left"
-msgstr ""
+msgstr "Prypnij do lewej"
```
Comment (rathann) [inline po/pl.po:133]: Typo.

File: po/pl.po [hunk 3]
```diff
@@ -126,23 +126,23 @@ msgstr "Położenie panelu na ekranie"
...
 
 #: prefs.js:444
 msgid "Stacked to bottom"
-msgstr ""
+msgstr "Prypnij do dołu"
```
Comment (rathann) [inline po/pl.po:137]: Typo.

File: po/pl.po [hunk 4]
```diff
@@ -126,23 +126,23 @@ msgstr "Położenie panelu na ekranie"
...
 
 #: prefs.js:444
 msgid "Stacked to right"
-msgstr ""
+msgstr "Prypnij do prawej"
```
Comment (rathann) [inline po/pl.po:141]: Typo.

File: po/pl.po [hunk 5]
```diff
@@ -126,23 +126,23 @@ msgstr "Położenie panelu na ekranie"
...
 
 #: prefs.js:445
 msgid "Centered"
-msgstr ""
+msgstr "Wyśrodkowano"
```
Comment (rathann) [inline po/pl.po:145]: Perhaps: "Wyśrodkowane"?

File: po/pl.po [hunk 6]
```diff
@@ -799,7 +799,7 @@ msgstr "Opóźnienie (ms) przed pokazaniem (domyślnie 100)"
 
 #: ui/BoxWindowPreviewOptions.ui.h:2
 msgid "(400 is default)"
-msgstr ""
+msgstr "(400 domyślnie)"
```
Comment (rathann) [inline po/pl.po:802]: In Polish, we usually put the number after the word in this context.

File: po/pl.po [hunk 7]
```diff
@@ -808,7 +808,7 @@ msgstr "Opóźnienie ukrywania miniatur (domyślnie 100 ms)"
 
 #: ui/BoxWindowPreviewOptions.ui.h:4
 msgid "(100 is default)"
-msgstr ""
+msgstr "(100 domyślnie)"
```
Comment (rathann) [inline po/pl.po:811]: In Polish, we usually put the number after the word in this context.

File: po/pl.po [hunk 8]
```diff
@@ -1283,11 +1272,11 @@ msgstr ""
 
 #: ui/SettingsFineTune.ui.h:18
 msgid "(e.g. date menu)"
-msgstr ""
+msgstr "(np. menu daty)"
 
 #: ui/SettingsFineTune.ui.h:19
 msgid "Force Activities hot corner on primary monitor"
-msgstr ""
+msgstr "Wymuś gorący róg na głównym monitorze"
```
Comment (rathann) [inline po/pl.po:1279]: Missing "Aktywności" here? I haven't checked the official GNOME translation for "Activities", though.

File: po/pl.po [hunk 9]
```diff
@@ -1321,17 +1310,17 @@ msgstr "Ukryj i odsłoń panel według preferencji"
 
 #: ui/SettingsPosition.ui.h:6
 msgid "Order and Position on monitors"
-msgstr ""
+msgstr "Kolejność i pozycja na monitorze"
```
Comment (rathann) [inline po/pl.po:1313]: The original is plural ("na monitorach").

File: po/pl.po [hunk 10]
```diff
@@ -1321,17 +1310,17 @@ msgstr "Ukryj i odsłoń panel według preferencji"
...
 #, fuzzy
 msgid "Monitor"
-msgstr "Monitor "
+msgstr "Monitor"
 
 #: ui/SettingsPosition.ui.h:8
 #, fuzzy
 msgid "Apply changes to all monitors"
-msgstr "Wyświetl panel na wszystkich monitorach"
+msgstr "Wyświetl na wszystkich monitorach"
```
Comment (rathann) [inline po/pl.po:1323]: Shouldn't it say: "Zastosuj zmiany na wszystkich monitorach"?

**Review by rathann — COMMENTED** [review]
Looks good, left some inline comments.

**Comment by alewicki95** [comment]
Yes, You are right @rathann, I made changes in file along with Your suggestions. Thank You.

**Comment by rathann** [comment]
Thanks for your contribution! You can simply push more commits to an existing PR (or even force-push) instead of closing and opening a new PR.

**Pull request closed without merging**